tests/ecosystem/upgrade/test_resources.py

```diff
@@ -90,3 +98,22 @@ def test_pod_io(
         helpers.wait_for_resource_state(pod, constants.STATUS_RUNNING, timeout=600)
         job_name = pod.get_labels().get("job-name")
         job_obj.delete(resource_name=job_name)
+
+
+@post_upgrade
+@bugzilla("1974343")
+@skipif_ocs_version("<4.4")
```

Comment:
Not sure if this will really matter, 4.4 is EOL and currently our production runs are triggered on n, n-1 atmax n-2 releases.